setup.py

```diff
@@ -5,6 +5,11 @@
 
 from setuptools import setup
 
+# Workaround for http://bugs.python.org/issue8876, see
+# http://bugs.python.org/issue8876#msg208792
+# This can be removed when using Python 2.7.9 or later:
+# https://hg.python.org/cpython/raw-file/v2.7.9/Misc/NEWS
+del os.link
```

Comment:
This is needed because distutils makes hard links, which are not supported in the virtual filesystem used by Vagrant's "synced folders". As a result, none of the `setup.py` commands work without this change.

Without `os.link`, the files are copied instead, which costs a little more storage, but the difference is marginal for the size files we're dealing with here. This workaround is recommended in the Python bug.

This issue was fixed in Python 2.7.9, but that is not the standard Python for Debian/Ubuntu right now so this seemed like the best solution for now.